crates/runtime/benches/bench_s3/mod.rs

```diff
@@ -63,28 +74,126 @@ pub(crate) async fn run(
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/income_band/",
+                "income_band",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/inventory/",
+                "inventory",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/store_sales/",
+                "store_sales",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/store_returns/",
+                "store_returns",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/web_sales/",
+                "web_sales",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/web_returns/",
+                "web_returns",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/customer/",
+                "customer",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/customer_address/",
+                "customer_address",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/customer_demographics/",
+                "customer_demographics",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/date_dim/",
+                "date_dim",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/household_demographics/",
+                "household_demographics",
+            ))
+            .with_dataset(make_dataset("spiceai-public-datasets/tpcds/item/", "item"))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/promotion/",
+                "promotion",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/reason/",
+                "reason",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/ship_mode/",
+                "ship_mode",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/store/",
+                "store",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/time_dim/",
+                "time_dim",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/warehouse/",
+                "warehouse",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/web_page/",
+                "web_page",
+            ))
+            .with_dataset(make_dataset(
+                "spiceai-public-datasets/tpcds/web_site/",
+                "web_site",
+            )),
+        _ => unreachable!(),
```

Comment:
```suggestion
        _ => panic!("Only tpcds or tpch benchmark suites are supported"),
```

add a message to this unreachable and make it a `panic!()` - it's not unreachable if someone just passes `-b not_a_real_benchmark` :smile: